### ISSUE TITLE
Crash trying to render items in JEI

### ISSUE THREAD

**Issue by Jumbumbum** [Asek3/Rubidium#553]
When searching or items specific ones crash the game, thought it was a create thing at first but then got the same thing with productive bees. https://pastebin.com/FWp3jJA9

**Comment by SiverDX** [comment]
problem stems from:

https://github.com/Creators-of-Create/Create/blob/104985d9ed521dde026e7be23d710858833b9578/src/main/java/com/simibubi/create/foundation/item/render/PartialItemModelRenderer.java#L67C8-L67C8

this might not be correctly called:

https://github.com/Asek3/rubidium-upstream/blob/053e35a31dbf86cfa2f25e4c2a4fea6756bfabbb/src/main/java/me/jellysquid/mods/sodium/mixin/features/shader/uniform/ShaderProgramMixin.java#L39

**Comment by drwoops** [comment]
https://github.com/Asek3/Rubidium/issues/550 is also about the same problem. we have started to see this issue frequently reported in the create mod discord/github.

**Comment by WilliamBlaze** [comment]
Yup, this looks exactly like the same issue I've been stumbling over recently, [as seen here](https://pastebin.com/5YZ9wiHb).
